bash_scripts/provision.sh

```diff
@@ -0,0 +1,318 @@
+#!/bin/bash
+set -e
+
+function init-variables {
+    local file="../env_variables"
+
+    CASSANDRA_REPLICATION_TYPE="Simple"
+    CASSANDRA_CONTACT_POINTS='172.16.238.68:9042' #initialized by the start-up script
```

Comment:
I wonder why you don't use DNS names instead of IP addresses.
You could use "cassandra-ms" instead of Cassandra IP. And similarly you could use "postgres-ms" instead of Postgres IP. The same goes with all other IP-s. If you could use DNS names instead of IP addresses then it would be more compatible with Mac as the Docker IP addresses don't work in Mac. Docker DNS addresses don't work either in Mac but a workaround for them is to add them to /etc/hosts and make them all point at 127.0.0.1.